### PR TITLE
Try to get the PR Reports workflow running again

### PR DESCRIPTION
**Pull request by chrisvest**
Motivation:
This workflow has been broken for a couple of months now.
It would be nice if we could get junit test reports back in out PR checks.

Modificatoin:
Fix a test matrix typo.

Result:
Hopefully the PR Reports workflow starts working again.

Draft until it actually does work. There are probably several things wrong that will take time to work through.

### CODE REVIEW

File: .github/workflows/ci-pr-reports.yml [hunk 1]
```diff
@@ -32,7 +32,7 @@ jobs:
           - setup: linux-x86_64-java11
           - setup: linux-x86_64-java16
           - setup: linux-x86_64-java11-boringssl
-          - setuo: windows-x86_64-java11-boringssl
+          - setup: windows-x86_64-java11-boringssl
```
Comment (normanmaurer) [inline .github/workflows/ci-pr-reports.yml:35]: 🤦

**Review by normanmaurer — APPROVED** [review]

**Comment by chrisvest** [comment]
Ok, looks like the PR build isn't actually running this action with the updated code, so we'll have to just merge it and see how it goes.

**Comment by normanmaurer** [comment]
@chrisvest please also cherry-pick into master

**Comment by chrisvest** [comment]
@normanmaurer Just pushed ;)

**Pull request merged**